src/app/Http/Controllers/BusinessTripController.php

```diff
@@ -77,4 +85,203 @@ public function confirm() {
+                    'department' => $trip->user->department,
+                    'place' => $trip->country->name . ', ' . $trip->place,
+                    'datetime_start' => $trip->datetime_start->format('Y-m-d'),
+                    'datetime_end' => $trip->datetime_end->format('Y-m-d'),
+                    'transport' => $trip->transport->name,
+                    'trip_purpose' => $trip->tripPurpose->name,
+                    'fund' => $trip->sppSymbol->fund,
+                    'functional_region' => $trip->sppSymbol->functional_region,
+                    'financial_centre' => $trip->sppSymbol->financial_centre,
+                    'spp_symbol' => $trip->sppSymbol->spp_symbol,
+                    'account' => $trip->sppSymbol->account,
+                    'grantee' => $trip->sppSymbol->grantee,
+                    'iban' => $trip->iban,
+                    'bank_name' => '?',
+                    'incumbent_name2' => '?', // sem sa bude dopĺnať meno a priezvisko z tabuľky staff
+                    'incumbent_name1' => '?', // -||-
+                    'position_name1' => '?',  // -||-
+                    'position_name2' => '?',  // -||-
+                    'contribution1_text' => '?', // v databáze neexistuje prepojenie na tabuľku s prínosmi pre fakultu
+                    'contribution2_text' => '?', // -||-
+                    'contribution3_text' => '?', // -||-
+                ];
+                break;
+
+            case DocumentType::CONTROL_SHEET:
+                $data = [
+                    'spp_symbol' => $trip->sppSymbol->spp_symbol,
+                    'expense_estimation' => $trip->expense_estimation,
+                    'incumbent_name1' => '?', // to isté čo v COMPENSATION_AGREEMENT, dopĺňanie z tabuľky staff
+                    'incumbent_name2' => '?', // -||-
+                    'incumbent_name3' => '?', // -||-
+                    'source1' => $trip->sppSymbol->fund, // niesom si istý, či zdroj je fund
+                    'source2' => $trip->sppSymbol->fund,
+                    'source3' => $trip->sppSymbol->fund,
+                    'functional_region1' => $trip->sppSymbol->functional_region,
+                    'functional_region2' => $trip->sppSymbol->functional_region,
+                    'functional_region3' => $trip->sppSymbol->functional_region,
+                    'spp_symbol1' => $trip->sppSymbol->spp_symbol,
+                    'spp_symbol2' => $trip->sppSymbol->spp_symbol,
+                    'spp_symbol3' => $trip->sppSymbol->spp_symbol,
+                    'financial_centre1' => $trip->sppSymbol->financial_centre,
+                    'financial_centre2' => $trip->sppSymbol->financial_centre,
+                    'financial_centre3' => $trip->sppSymbol->financial_centre,
+                    'incumbent_name4' => '?', // to čo vyššie
+                    'incumbent_name5' => '?',
+                    'purpose_details' => $trip->purpose_details,
```

Comment:
ako pozeram na to pdf, tak toto by mal byt ten "popis pozadovaneho tovaru....". ak ano, tak tam ma byt napisane iba "Úhrada vložného"